configure.ac

```diff
@@ -1538,15 +1538,6 @@ AC_ARG_WITH(pidfile,
 MASTERPIDFILE="\"$MASTERPIDFILE\""
 AC_DEFINE_UNQUOTED(MASTER_PIDFILE, $MASTERPIDFILE,[Name of the pidfile for master])
 
-dnl
-dnl see if we're compiling with autocreate support
-dnl
-AC_ARG_ENABLE(autocreate,
-        [AS_HELP_STRING([--enable-autocreate], [enable autocreate support])],,[enable_autocreate="no";])
```

Comment:
Rather than removing all this, I think I'd prefer if we just flipped the default to "yes".  That way anyone who maybe doesn't trust the autocreate code (reasonable!) can still configure with `--disable-autocreate` and not have to think about it.

Longer term, removing a bunch of conditional compilation is one of our goals, because it reduces the size of the matrix of configurations that hypothetically need testing.  But the first step towards that is including it by default, not including it unconditionally. :)